sysdig/resource_sysdig_secure_cloud_auth_account.go

```diff
@@ -387,26 +387,42 @@ func constructAccountComponents(data *schema.ResourceData) []*cloudauth.AccountC
 					if data.Get(SchemaCloudProviderType).(string) == cloudauth.Provider_PROVIDER_GCP.String() {
 						spGcp := &internalServicePrincipalMetadata{}
 						err = json.Unmarshal([]byte(value.(string)), spGcp)
-						if len(spGcp.Gcp.Key) >= 0 {
```

Comment:
`>=` is a legitimate, but the only bug here. Should be `>`, which I'd like to see be the only change within this section